pages/common/uniq.md

```diff
@@ -1,23 +1,23 @@
 # uniq
 
-> Output the unique lines from the given input or file.
+> Output the unique lines from the given input or file. Since it does not detect repeated lines unless they are adjacent, so we need to sort them first.
```

Comment:
Please add the second sentence in a new line, and remove the "so", which is redundant with "Since".